src/TagLib/Ogg/XiphComment.cs

```diff
@@ -48,17 +48,23 @@ public class XiphComment : TagLib.Tag, IEnumerable<string>
 			new Dictionary<string, string[]> ();
 
 		/// <summary>
-		///    Contains the ventor ID.
+		///    Contains the vendor ID.
 		/// </summary>
 		private string vendor_id;
 
+		/// <summary>
+		///    Saves BeatsPerMinute tag as either "Tempo" or "BPM"
+		///    based on which was last read.
+		/// </summary>
+		private static bool save_BPM_as_tempo = true;
```

Comment:
This is very `c` like and not very csharp-y. Maybe `SaveBeatsPerMinuteAsTempo`?

Reply:
Yes, at work we're using c++, so that's probably why...
Updated.